apps/codeforafrica/src/components/NavList/NavListItem.js

```diff
@@ -0,0 +1,93 @@
+import { styled } from "@mui/material/styles";
+import PropTypes from "prop-types";
+import * as React from "react";
+
+const NavLinkGrid = styled("li")(({ theme: { breakpoints, typography } }) => ({
+  margin: `${typography.pxToRem(20)} 0`,
+  listStyle: "none",
+  [breakpoints.up("lg")]: {
+    margin: `${typography.pxToRem(10)} 0`,
+    listStyle: "none",
+  },
+}));
+
+const NavLink = styled("a", {
```

Comment:
I think we can just use `@commons-ui/next/Link` for this. Since Link supports all `Typography` styles, one should be able to easily customize it using `sx` prop e.g.
```jsx
  <Link sx={{ bgColor: 'inherit', typography: 'body1' }}
```
For it to take body1 sizes and inherit the background it's been put on (white for navigation (desktop), blue for navigation (mobile), and black for footer).